packages/trader/src/Modules/Trading/Components/Form/TradeParams/strike.jsx

```diff
@@ -120,7 +123,9 @@ const Strike = observer(() => {
             <MobileWrapper>
                 <div className='mobile-widget__wrapper'>
                     <div className='strike-widget' onClick={toggleWidget}>
-                        <div className='mobile-widget__spot'>{<Text size='xs'>{localize('Spot')}</Text>}</div>
```

Comment:
Question: why the logic here is different from above, and why don't you use `is_relative_strike_applicable` do display it our not?

Reply:
Because I check `is_24_hours_contract` and it gives me 'false' even if I choose 1 day contract, but I didn't check `is_relative_strike_applicable`. Sorry, will check and add

Reply:
Hmm, if I choose 24 hours (not 1 day) `is_relative_strike_applicable` is the same as for 1 day contract. So if I use it I'll also need to add check if it's day contract or not, because 'Spot' should be shown for hours contract (even if its 24 hours)

Reply:
In other words:
'Spot' is shown for minutes/hours. Not shown for days.
If I use is_relative_strike_applicable I'll not show 'Spot' for days and for hours contract > 24